diagnostic.md

```diff
@@ -6,73 +6,102 @@ What is the purpose of a backend?
 
 ```md
 // your response here
+The purpose of a backend is to allow for CRUD commands: creating, reading, updating, and deleting data. It helps us communicate with stored data and returns them to the viewer.
```

Comment:
Good it allows our data to persist. It also enables multiple users to interact with each other from different devices while on the same app.